### DISCUSSION TITLE
Difference between LSPosed-riru and LSPosed-zygisk ?

### DISCUSSION THREAD

**Discussion by xerta555** [LSPosed/LSPosed#1340] [Q&A] [answered]
Hi

I see that since new Magisk canary with Zygisk has been released, 2 types of LSPosed zip installer is available:
- LSPosed-zygisk
- LSPosed-riru

But I had read that Zygisk will not be compatible with LSPosed.
So what's the difference(s) between theses 2 types of LSPosed?

**Comment by yujincheng08** [comment, marked as answer]
> But I had read that Zygisk will not be compatible with LSPosed.

FAKE NEWS!!!

**Reply by xerta555** [reply]
> 
> 
> > But I had read that Zygisk will not be compatible with LSPosed.
> 
> FAKE NEWS!!!

Ah ok sorry, so if i understand the first one is compatible with Zygisk and the second one is for Magisk versions before Zygisk implementation?

**Reply by yujincheng08** [reply]
Why didn't you read our channels' post:
https://t.me/LSPosed/139
https://t.me/LSPosed/140
https://t.me/LSPosed/145
